docs/guide/migration.md

```diff
@@ -21,7 +21,7 @@ test('validation works', { retry: 3 }, () => { // [!code ++]
 })
 ```
 
-Vitest 4.0 will throw an error if the third argument is an object. Note that the timeout number is not deprecated:
+Vitest 3.0 will throw an error if the third argument is an object. Note that the timeout number is not deprecated:
```

Comment:
Vitest 3 is out and it doesn't throw an error, it prints a warning (see paragraph above)

Vitest 4 (the next major) however will throw an error

Reply:
@sheremet-va Oh, I missed that point. It should be the next major version that throws an error.

Should I change it to:
`Vitest 4.0 (the next major version) will throw an error if the third argument is an object.`
or
`The next major version will throw an error if the third argument is an object.`

I think specifying only `the next major version` would make it clearer than including a specific version number.

Reply:
Sure